### ISSUE TITLE
types for xDomain

### ISSUE THREAD

**Issue by MCMattia** [mhkeller/layercake#117]
Hi guys, I have been using your library to make a simple 3 line graph. It works very good. Thank you for your work.

I'm using yDomain in the following way:
`yDomain={[0, null]}`

Since the types for yDomain are:
`[min: ?Number, max: ?Number]`
Typescript is warning that `null` is not assignable to a number.

I think that the types for xDomain or yDomain, should be:
`Array:[min: ?Number|null, max: ?Number|null]|String[]|Number[]|Function`
instead of:
`Array:[min: ?Number, max: ?Number]|String[]|Number[]|Function`

Basically adding the null option since as you document on your website:
_If you want to inherit the value from the data's extent, set that value to null._

**Comment by mhkeller** [comment]
Great catch – thanks! I'll put in a fix for this.

**Comment by mhkeller** [comment]
@MCMattia this is done in [this branch](https://github.com/mhkeller/layercake/tree/svelte-package-migration). Svelte kit has updated the way that it packages libraries and it requires changing the deployment and packing system, which I won't be able to do today. I need to make sure it doesn't mess a bunch of things up. Hopefully it won't be too much trouble and you can add a ts ignore for now.

**Comment by MCMattia** [comment]
Hi @mhkeller thank you for this work. For now I have manually fixed it in the `LayerCake.svelte.d.ts` file

**Comment by mhkeller** [comment]
This is updated in 7.2.3
